cli/src/test/java/de/jplag/cli/LanguageSuffixConflictsTest.java

```diff
@@ -0,0 +1,14 @@
+package de.jplag.cli;
+
+import org.junit.jupiter.api.Test;
+
+import de.jplag.multilang.MultiLanguageOptions;
+import de.jplag.multilang.MultiLanguageParser;
+
+public class LanguageSuffixConflictsTest {
+    @Test
+    void testNoConflictsBetweenLanguageSuffixes() {
+        new MultiLanguageParser(new MultiLanguageOptions()); // triggers internal check for conflicts automatically.
+        // For conflicting language, see error message
+    }
```

Comment:
Use `assertDoesNotThrow(() -> new MultiLanguageParser(new MultiLanguageOptions()), message)` here to make it a proper test case. For the message, write a descriptive message that someone in 10 years understands and has no clue about the underlying assumptions of the multi-language module. For example, along the lines of "There is conflict regarding file extensions between two or more language modules. Ensure exactly one module has priority."

Reply:
As discussed in the meeting, it is enough, just use my wording in the exception.